packages/multi-select-combo-box/src/vaadin-lit-multi-select-combo-box.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license
+ * Copyright (c) 2021 - 2024 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import './vaadin-lit-multi-select-combo-box-chip.js';
+import './vaadin-lit-multi-select-combo-box-container.js';
+import './vaadin-lit-multi-select-combo-box-internal.js';
+import { html, LitElement } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { inputFieldShared } from '@vaadin/field-base/src/styles/input-field-shared-styles.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { MultiSelectComboBoxMixin } from './vaadin-multi-select-combo-box-mixin.js';
+import { multiSelectComboBox } from './vaadin-multi-select-combo-box-styles.js';
+
+/**
+ * LitElement based version of `<vaadin-multi-select-combo-box>` web component.
+ *
+ * ## Disclaimer
+ *
+ * This component is an experiment and not yet a part of Vaadin platform.
+ * There is no ETA regarding specific Vaadin version where it'll land.
+ * Feel free to try this code in your apps as per Apache 2.0 license.
+ */
+class MultiSelectComboBox extends MultiSelectComboBoxMixin(ThemableMixin(ElementMixin(PolylitMixin(LitElement)))) {
+  static get is() {
+    return 'vaadin-multi-select-combo-box';
+  }
+
+  static get styles() {
+    return [inputFieldShared, multiSelectComboBox];
+  }
+
+  /** @protected */
+  render() {
+    return html`
+      <div class="vaadin-multi-select-combo-box-container">
+        <div part="label">
+          <slot name="label"></slot>
+          <span part="required-indicator" aria-hidden="true" @click="${this.focus}"></span>
+        </div>
+
+        <vaadin-multi-select-combo-box-internal
+          id="comboBox"
+          .filteredItems="${this.filteredItems}"
+          .items="${this.items}"
```

Comment:
Note: these properties need to come in the specific order, otherwise if `filteredItems` is after `items` it overrides the `filteredItems` in the internal combo-box (seems to be related to how Lit sets properties).